tests/framework/PostProcessors/InterfacedPostProcessor/test_interfacedPP.xml

```diff
@@ -23,18 +23,17 @@
 
   <Steps>
     <MultiRun name="FirstMRun" pauseAtEnd="True">
-      <Input class="DataObjects" type="PointSet">inputPlaceHolder</Input>
-      <Model class="Models" type="ExternalModel">PythonModule</Model>
-      <Sampler class="Samplers" type="MonteCarlo">MC_external</Sampler>
-      <Output class="DataObjects" type="PointSet">outMC</Output>
-      <Output class="Databases" type="HDF5">out_db</Output>
-      <Output class="OutStreams" type="Print">PrintHistorySet_dump</Output>
+      <Input   class="DataObjects" type="PointSet"      >inputPlaceHolder</Input>
```

Comment:
This formatting will all get erased as soon as a conversion script is run.  I highly recommend against adding it.

I realize it's nice to see lined up, but I think we should leave that to a proper UI, not regression test files.